zipper/unzipper.cpp

```diff
@@ -192,7 +192,7 @@ struct Unzipper::Impl
 
     void changeFileDate(const std::string& filename, uLong dosdate, tm_unz tmu_date)
     {
-#ifdef _WIN32
+#if (USE_WINDOWS)
```

Comment:
should not be #if defined(USE_WINDOWS) ?

Reply:
Sorry, i didn't browse the repository too much so i didn't notice that the preferred method was `#if defined(USE_WINDOWS)`